cmd/ledgerutil/main.go

```diff
@@ -26,6 +26,7 @@ var (
 	snapshotPath1 = compare.Arg("snapshotPath1", "First ledger snapshot directory.").Required().String()
 	snapshotPath2 = compare.Arg("snapshotPath2", "Second ledger snapshot directory.").Required().String()
 	outputDir     = compare.Flag("outputDir", "Snapshot comparison json results output directory. Default is the current directory.").Short('o').String()
+	first         = compare.Flag("first", "Generates additional json output containing first n differences ordered by height, where n is provided as parameter.").Short('f').Int()
```

Comment:
I thought we decided the default would be 10. Defaults can be set on flags with .Default().

Reply:
The problem with the Default() function is that it will automatically set the respective first variable to 10, regardless of if the user used the first flag or not. In this case, there is no way to differentiate between the user wanting to use the first flag on default or not wanting to use the flag at all.

Reply:
I think we should always report the first diffs. If user doesn't pass flag, report the first 10 diffs.
We don't want somebody to run the utility and provide the results to support, only for support to have to tell them to run it again with first diffs set.
Only if they pass '0' should we skip it.